aten/src/ATen/native/Cross.cpp

```diff
@@ -1,52 +1,59 @@
 #include <ATen/ATen.h>
 #include <ATen/Dispatch.h>
 #include <ATen/NativeFunctions.h>
+#include <ATen/native/Resize.h>
+#include <ATen/ExpandUtils.h>
```

Comment:
Nit. Is this import used?

Reply:
I included this for `infer_size` but removing `#include <ATen/ExpandUtils.h>` is ok.

Reply:
I hadn't realised that `inver_size` needs `ExpandUtils.h`. Then leave it, every file should be self-consistent and not depend on what other files include or not.